fem/pbilinearform.cpp

```diff
@@ -295,6 +295,39 @@ const
    pfes->Dof_TrueDof_Matrix()->MultTranspose(a, Y, 1.0, y);
 }
 
+double ParBilinearForm::InnerProduct(const ParGridFunction &x,
+                                     const ParGridFunction &y)
+{
+   MFEM_ASSERT(x.ParFESpace() == pfes, "the parallel spaces must agree");
```

Comment:
Maybe use the word "match" instead of "agree"?

Reply:
Ok, that is maybe better.